tox.ini

```diff
@@ -27,11 +27,18 @@ setenv =
     PYTHONPATH = {toxinidir}
 extras = dev
 download = true
-install_command = python -m pip install --no-user {opts} {packages}
+install_command =
+    python -m pip install --no-user {opts} {packages}
+    python -m pip install --no-user {opts} {packages} --install-option="--with-binaries"
```

Comment:
Why twice the same command ? 
Don't we want to run tests inside the conda environment ?

Reply:
The reason for calling the command twice is due to the fact that calling `--install-option` during a pip installation disables `wheels`. For most of the dependencies we have, `wheels` are preferable, so installing all those are done in the first call, and compiling raven/ostrich is performed in the second.

Reply:
But I believe the second command won't install ravenpy since it's already installed. You need to uninstall ravenpy before installing it with the binaries.